platform/linux-dpdk/Makefile.am

```diff
@@ -196,6 +197,7 @@ noinst_HEADERS = \
 		  ${top_srcdir}/platform/linux-generic/include/odp_packet_socket.h \
 		  ${top_srcdir}/platform/linux-generic/include/odp_pkt_queue_internal.h \
 		  ${srcdir}/include/odp_pool_internal.h \
```

Comment:
currently they are differentiated with platform/linux-*. Unless if we do any directory re-structuring, I assume this is sufficient.